app/models/gobierto_data/connection.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require_dependency "gobierto_data"
+
+module GobiertoData
+  class Connection < ActiveRecord::Base
+    self.abstract_class = true
+
+    class << self
+      def execute_query(site, query)
+        base_connection_config = connection_config
+        return null_query unless (db_config = site&.gobierto_data_settings&.db_config&.dig(Rails.env)).present?
```

Comment:
Why Rails.env here?

Reply:
Yes, maybe this is unnecessary. I used the configuration hash generated by `config/database.yml`, with the option to have different configurations for each environment, but since each configuration on each site may be different, the environment can be useless. I'll change it